### PR TITLE
Support Rails 5

### PR DESCRIPTION
**Pull request by dlackty**
Basically follow the commit modosc/cloudflare-rails@7a52ca325fcb19fb54a7f7ecc385ad5784957031

### CODE REVIEW

**Comment by dinks** [comment]
Could you may be update the `README.md` file?

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/cloudfront-rails. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+Bug reports and pull requests are welcome on GitHub at https://github.com/modosc/cloudfront-rails. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
```
Comment (dinks) [inline README.md]: `https://github.com/dinks/cloudfront-rails`

**Comment by dinks** [comment]
One small change and we are good to go!

File: README.md (hunk 1 above)
Comment (dlackty) [inline README.md]: Absolutely a stupid mistake. Now corrected.

**Comment by dinks** [comment]
Thanks for the support @dlackty ! The gem has been released.

**Pull request merged**